it/uk/gov/hmrc/incorporatedentityidentificationfrontend/api/controllers/JourneyControllerISpec.scala

```diff
@@ -234,6 +235,200 @@ class JourneyControllerISpec extends ComponentSpecHelper with JourneyStub with I
         )
       }
     }
+    "return captured data" when {
+      "the journeyId exists and verificationStatus is BusinessVerificationFail" in {
+        stubAuth(OK, successfulAuthResponse(Some(testInternalId)))
+        stubRetrieveIncorporatedEntityInformation(testJourneyId)(
+          status = OK,
+          body = Json.toJsObject(
+            IncorporatedEntityInformation(
+              companyProfile = testCompanyProfile,
+              optCtutr = Some(testCtutr),
+              identifiersMatch = false,
+              businessVerification = Some(BusinessVerificationFail),
+              registration = testSuccessfulRegistration
+            )
+          )
+        )
+
+        lazy val result = get(s"/incorporated-entity-identification/api/journey/$testJourneyId")
+
+        result.status mustBe OK
+
+        result.json mustBe Json.obj(
```

Comment:
Instead of repeating a large Json object you could write a method to create the json with a few input parameters. This will be easier when business verification and registration status are simple strings.

Reply:
you know that I am still laughing ;-)